src/scripts/core/axis/y-axis.js

```diff
@@ -19,7 +19,7 @@
             var domain = this.domain;
             var dMin = options.min != null ? options.min : options.zeroAnchor ? Math.min(0, domain[0]) : domain[0];
             var dMax = options.max != null ? options.max : domain[1];
-            var tickValues = options.tickValues || _.nw.niceTicks(dMin, dMax, options.ticks);
+            var tickValues = options.tickValues || _.nw.niceTicks(dMin, dMax, options.ticks, options.zeroAnchor, options);
```

Comment:
all functions should be given _only_ information they need. Passing the entire `options` object to niceticks gives it tmi.